### ISSUE TITLE
niconico | sp.nicovideo.jp

### ISSUE THREAD

**Issue by Slasar41** [PreMiD/Presences#2422]
**Prerequisites and essential questions** 
- [x] Is it a popular site?
- [x] Is the website older than 2 months? 
- [ ] Is the site locked to a specific country/region?
- [ ] Is the site a paid service? (e.g. Netflix, Hulu)
- [ ] Does the website feature NSFW content? (e.g. porn, etc...)
- [ ] Are you a donator/patron?
- [x] Do you acknowledge that coding presences is completely voluntary and may take time for your service to be added regardless of priority?

**What's your Discord username?** 
Slasar41#8399
**What's the name of the service?** 
niconico | sp.nicovideo.jp
**What should the Presence display?** 
Basically it's the same as www.nicovideo.jp but you watch the video full screen with a flying comment text.
**If possible, please provide a logo for the service (512x512 minimum)**

**Comment by ririxichan** [comment]
Please upload logo

**Comment by Slasar41** [comment]
Do I have to?
The niconico presence is already supported with the logo, I just want to request another subdomain to be added to niconico presence.

**Comment by ririxichan** [comment]
Oh okey. so you could do feature request instead presence request? Thanks.
